compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -33,6 +33,8 @@ object Parsers {
   import reporting.Message
   import reporting.messages._
 
+  val AllowOldWhiteboxSyntax = true
```

Comment:
This seems to be unnecessary.

Reply:
This is so that someone can come back and remove the old syntax